api/src/main/resources/org/openmrs/module/isanteplusreports/sql/healthQualReports/earlyInfantDiagnosis.sql

```diff
@@ -0,0 +1,78 @@
+SELECT
+	COUNT( DISTINCT CASE WHEN (
+        p.gender = 'F'
+            AND p.patient_id IN (
+                SELECT plab.patient_id
+                FROM isanteplus.patient_laboratory plab
+                WHERE
+                    plab.test_done = 1
+                    AND plab.test_id = 844
```

Comment:
if such values are made configurable, will allow flexibility, for example wrapping all that into a method and pass the arguments then evaluate them at cohort definition

Reply:
I agree. 
However, the main aim of this process was to ensure that the reports are pulling out correct data as defined in the indicator definition document.
I shall plan to refactor this code later based on the work I shall carry out on the other indicator reports.